test/reactionsystem_core/reactionsystem.jl

```diff
@@ -118,13 +118,11 @@ let
     odesys = complete(convert(ODESystem, rs))
     sdesys = complete(convert(SDESystem, rs))
     js = complete(convert(JumpSystem, rs))
-    nlsys = complete(convert(NonlinearSystem, rs))
```

Comment:
For this and the next one. Here we test the conversion of a NonlinearSystem that is non-autonomous, which we no longer permit. Hence these tests were removed. I marked the second one as broken since I am not sure if there is some important test here or not.